### ISSUE TITLE
className on ShareButtons doesn't work

### ISSUE THREAD

**Issue by kyouko-taiga** [nygardk/react-share#4]
The className attribute of ShareButtons doesn't seem to have any effect on the rendered component.

Here is a snippet extracted from my code that illustrates the problem. First my jsx:
```javascript
<GooglePlusShareButton
    url="http://www.google.ch/"
    className="my-awesome-class">
    <Button>
        <i className="fa fa-fw fa-lg fa-google-plus" />
    </Button>
</GooglePlusShareButton>
```
And the rendered component:
```html
<div class="SocialMediaShareButton SocialMediaShareButton--google-plus" data-reactid=".0.2.0.1.2.0:$366.1:$0.0.1.4.2">
    <button type="button" class="btn btn-default" data-reactid=".0.2.0.1.2.0:$366.1:$0.0.1.4.2.0">
        <i class="fa fa-fw fa-lg fa-google-plus" data-reactid=".0.2.0.1.2.0:$366.1:$0.0.1.4.2.0.0"></i>
    </button>
</div>
```

The same problem also appears in the demo code. I checked on both Chrome (47.0.2526.106, x64) and Safari (9.0.2).

**Comment by nygardk** [comment]
Thanks for pointing out this issue! I fixed and released it immediately in version 1.4.1.

**Comment by kyouko-taiga** [comment]
Thanks!
